docs/general/trace.md

```diff
@@ -36,4 +35,45 @@ Apart from semantic conventions for traces, [metrics](metrics.md), [logs](logs.m
 OpenTelemetry also defines the concept of overarching [Resources](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.51.0/specification/resource/sdk.md) with their own
 [Resource Semantic Conventions](/docs/resource/README.md).
 
+## Compatibility
+
+This section defines trace semantic conventions used by the
+compatibility components, e.g. OpenTracing Shim layer.
+
+### OpenTracing
```

Comment:
I'm not sure I agree with moving OpenTracing here to such an important location

Unlike others which are about compatibility with other actively maintained projects in the ecosystem, this is about backwards compatibility with a legacy project.